### PR TITLE
Yaz.update bandwidth requirement

### PR DESCRIPTION
**Pull request by YazzyYaz**
Resolves this ticket: https://github.com/celestiaorg/docs/issues/42

### CODE REVIEW

File: docs/nodes/light-node.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ The following hardware minimum requirements are recommended for running the ligh
 * Memory: 8 GB RAM
 * CPU: Quad-Core
 * Disk: 5 GB SSD Storage
-* Bandwidth: 1 GB of input-output connection
+* Bandwidth: 1 Gbps for Download/100 Mbps for Upload
```
Comment (adlerjohn) [inline docs/nodes/light-node.md:20]: Is this actually needed for a light node? I guess it's just copy-pasted for now, but...the requirements here should be a cell phone and dial-up!

Comment (liamsi) [reply docs/nodes/light-node.md:20]: I think reality is that DASing plus maintaining connections to sufficient fullnodes is taxing on the bandwidth requirements.

Comment (adlerjohn) [reply docs/nodes/light-node.md:20]: @YazzyYaz the requirements should be something like 2 GB or RAM and 56 kbps up and down.

Comment (YazzyYaz) [reply docs/nodes/light-node.md:20]: ok sounds good, will fix

Comment (YazzyYaz) [reply docs/nodes/light-node.md]: @adlerjohn updated the doc

File: docs/nodes/light-node.md [hunk 2]
```diff
@@ -14,10 +14,10 @@ Light Nodes have the following properties:
 
 ## Hardware Requirements
 The following hardware minimum requirements are recommended for running the light node:
-* Memory: 8 GB RAM
-* CPU: Quad-Core
+* Memory: 2 GB RAM
+* CPU: Single Core
 * Disk: 5 GB SSD Storage
-* Bandwidth: 1 GB of input-output connection
+* Bandwidth: 1 56 Kbps for Download/56 Kbps for Upload
```
Comment (adlerjohn) [inline docs/nodes/light-node.md:20]:
```suggestion
* Bandwidth: 56 Kbps for Download/56 Kbps for Upload
```

**Review by adlerjohn — APPROVED** [review]
:shipit:

**Review by Wondertan — APPROVED** [review]

**Review by liamsi — APPROVED** [review]

**Pull request merged**